butterfly/hdf5.py

```diff
@@ -56,17 +56,17 @@ def loadFolder(self,path):
                     d[K_DEPTH] = fd[d[K_DATASET_PATH]].shape[0]
                     self._dtype = fd[d[K_DATASET_PATH]].dtype
             return result
-                        
+
         elif path.endswith('.h5'):
             with h5py.File(path, "r") as fd:
                 key0 = fd.keys()[0]
                 self._dtype = fd[key0].dtype
-                return {
+                return [{
                     K_FILENAME : path,
                     K_DATASET_PATH : key0,
                     K_DEPTH: fd[key0].shape[0],
                     K_Z_OFFSET: 0,
-                }
+                }]
```

Comment:
The loadFolder method must return an array for congruity with the case where multiple members of the returned array are used to represent slices in the volume.